src/utils/components/Consoles/components/vnc-console/VncConsole.tsx

```diff
@@ -52,8 +53,18 @@ export const VncConsole: React.FC<VncConsoleProps> = ({
   const [status, setStatus] = React.useState<ConsoleState>(disconnected);
   const staticRenderLocaitonRef = React.useRef(null);
   const StaticRenderLocaiton = React.useMemo(
-    () => <div id={consoleContainerId} ref={staticRenderLocaitonRef} />,
-    [staticRenderLocaitonRef, consoleContainerId],
+    () => (
+      <div
+        style={{
+          height: '100%',
+          width: '100%',
+          display: !(status === connected) ? 'none' : 'block',
```

Comment:
why not status !== connected?
and maybe create a `className` instead of using `style`?

Reply:
u right, i changed this to a different solution